README.md

```diff
@@ -32,6 +32,13 @@ Steps to work on this project
 
 ![](user-guide.PNG?raw=true)
 
+Regarding the goal dashbord:
```

Comment:
Kan vi give den her en header, så den følger stilen? 
Altså, i stedet for "regarding goal dashboard" så skrive "#### Goal dashboard"